### DISCUSSION TITLE
Full Text Search Engine with Blazor FluentUI

### DISCUSSION THREAD

**Discussion by bytefish** [microsoft/fluentui-blazor#2463] [Show and tell]
I have written a Full Text Search Engine, that uses Blazor with FluentUI for its Frontend:

* https://github.com/bytefish/ElasticsearchFulltextExample

The example shows, among many other things, how to do File Uploads with the FluentUI Components and contains a component for Google-like Search Suggestions, that may be interesting. It’s also dockerized, so it shows how to containerize a Blazor WASM application.

@vnbaaij As some kind of feedback, because it turned out harder than expected to recreate the Angular Material version in Blazor FluentUI. It would be great to have a Token Input Field (for example to add tags) and a way to configure a Suggestion List for the Search component (or build a separate component). I’ve tried to use the Autocomplete component and Search with a ListBox, but failed with both.

**Comment by vnbaaij** [comment]
Great showcase (again!). The token input field looks like a good candidate for adoption.
